util/dvsim/Launcher.py

```diff
@@ -150,6 +150,9 @@ def __init__(self, deploy):
         # create a new one.
         self.renew_odir = False
 
+        # Store the CPU time from the run log.
+        self.cpu_time = ""
```

Comment:
How about `wall_clock_time` or `job_runtime`?

Reply:
Thank you! Updated to use `job_runtime`!